future-apps/payroll/contracts/Payroll.sol

```diff
@@ -601,8 +600,8 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
 
                 // Convert amount (in denomination tokens) to payout token and apply allocation
                 uint256 tokenAmount = _totalAmount.mul(exchangeRate).mul(tokenAllocation);
-                // Divide by 100 for the allocation percentage and by ONE for the exchange rate precision
-                tokenAmount = tokenAmount / (100 * ONE);
+                // Divide by 100 for the allocation percentage and by the exchange rate precision
+                tokenAmount = tokenAmount.div(100).div(feed.ratePrecision());
```

Comment:
Actually, another thought on this, we could cache this value before going into the for loop since it'll always be the same.

Reply:
sure, will do that in the last PR to avoid triggering the CI again 😓